### DISCUSSION TITLE
Как управлять сервисом после установки на роутер c OpenWrt?

### DISCUSSION THREAD

**Discussion by lsahnicne** [bol-van/zapret#290] [Q&A] [answered]
Понимаю, что вопрос дурацкий, но тем не менее.
Установил, следуя инструкции в [quick_start.txt](https://github.com/bol-van/zapret/blob/master/docs/quick_start.txt). Всё работает, однако не знаю как в случае необходимости остановить или перезапустить программу (например, если потребуется внести изменения в config).

**Comment by bol-van** [comment, marked as answer]
/etc/init.d/zapret start|stop
service zapret start|stop

**Comment by BubaMuba** [comment]
У меня тоже есть дурацкий вопрос: заметил, что обновляются и бинарники и скрипты. Как правильней обновляться: менять вручную или через uninstall -> install?

**Reply by bol-van** [reply]
Если есть много места и git, можно через git pull
Иначе льем zip с гитхаба в /tmp и прогоняем инсталятор с удалением /opt/zapret и копированием новых файлов (инсталятор это сам сделает). Конфиги и другие основные изменяемые файлы сохраняются
